.github/workflows/build.yaml

```
@@ -0,0 +1,26 @@
+name: Compile And Test
+on:
+  push:
+    branches:
+      - "**"
+jobs:
+  build-and-deploy:
```

nit: it's not deploying
```suggestion
  build-and-test:
```